monitor.cpp

```diff
@@ -1,38 +1,41 @@
 #include "./monitor.h"
 #include <assert.h>
+#include <iostream>
 #include <thread>
 #include <chrono>
-#include <iostream>
-using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+#include <string>
```

Comment:
This is needed in the header file, since the type `string` appears in the function signature